core-bundle/assets/scripts/tips.js

```diff
@@ -76,9 +78,20 @@
                 timer = setTimeout(function() {
                     tip.style.display = 'none';
                     tip.style.willChange = 'auto';
-                }, 100)
+                }, 'mouseleave' === e.type ? 100 : 0)
+            }
+        }
+
+        el.addEventListener('mouseleave', close);
+
+        // Close tooltip when touching anywhere else
+        document.addEventListener('touchend', (e) => {
+            if (e.originalTarget.contains(el)) {
```

Comment:
```suggestion
            if (e.target.contains(el)) {
```
I think `originalTarget` is not supported everywhere and not needed here?

Reply:
Hm, I think it needs to be `el.contains(e.target)` anyway, doesn't it? I always get confused here 😁

Reply:
Yes ☺️

```suggestion
            if (el.contains(e.target)) {
```